tools/validator.py

```diff
@@ -104,20 +184,18 @@ def CheckFile(self, filename):
                   artifact_definition.name, filename))
           result = False
 
+        artifact_definition_supports_windows = (
+            definitions.SUPPORTED_OS_WINDOWS in (
+                artifact_definition.supported_os))
+
         for source in artifact_definition.sources:
           if source.type_indicator in (
               definitions.TYPE_INDICATOR_FILE, definitions.TYPE_INDICATOR_PATH):
-            if definitions.SUPPORTED_OS_WINDOWS in source.supported_os:
+            if (artifact_definition_supports_windows or
```

Comment:
Why not an and instead of a nested if?

Reply:
Mainly because I'm planning to add other checks e.g. https://github.com/ForensicArtifacts/artifacts/issues/309